### DISCUSSION TITLE
Add/Edit link in WYSIWYG, allow linking to files

### DISCUSSION THREAD

**Discussion by PeterSchoell** [directus/directus#16505] [Feature Requests]
Request for integration of Add/Edit Link function in WYSIWYG Editor. 
This function was still available in v8, unfortunately no longer in v9
 
![image](https://user-images.githubusercontent.com/48986468/202528814-aa583142-e4e2-4c9f-a798-9ddee710d9fb.png)

The old function in the link dialog was sufficient and should definitely be integrated again.

_Originally posted by @PeterSchoell in https://github.com/directus/directus/discussions/6605#discussioncomment-4170529_

**Comment by Leapfrognz** [comment]
+1 for this feature. Very important to let authors create links to files in WYSIWYG editors - IMO a fundamental feature for any CMS.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
